backend/resolvers/types/otherMediaPieceType.js

```diff
@@ -0,0 +1,10 @@
+import { PieceBase } from './pieceType'
```

Comment:
This is a type definition for Apollo, makes sure the Piece grabs it's subtype components for Other Media (there's one of these for Photos and Videos too).